solution/src/main/openapi/solution.yaml

```diff
@@ -308,54 +308,124 @@ paths:
         required: true
         schema:
           type: string
-    patch:
-      operationId: updateSolutionParameterGroups
+    post:
+      operationId: createSolutionParameterGroup
       tags:
         - solution
-      summary: Update solution parameter groups
+      summary: Create a solution parameter group
       requestBody:
-        description: Parameter groups to update
+        description: Parameter groups to create
         required: true
         content:
           application/json:
             schema:
-              type: array
-              items:
-                $ref: "#/components/schemas/RunTemplateParameterGroup"
-            examples:
-              ParameterGroups:
-                $ref: '#/components/examples/ParameterGroups'
+              $ref: "#/components/schemas/RunTemplateParameterGroupCreateRequest"
           application/yaml:
             schema:
-              type: array
-              items:
-                $ref: "#/components/schemas/RunTemplateParameterGroup"
-            examples:
-              ParameterGroups:
-                $ref: '#/components/examples/ParameterGroups'
+              $ref: "#/components/schemas/RunTemplateParameterGroupCreateRequest"
       responses:
         "201":
-          description: Parameter groups successfully updated
+          description: Parameter groups successfully created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RunTemplateParameterGroup'
+        "400":
+          description: Bad request - Invalid parameter group
+        "404":
+          description: Solution not found or insufficient access rights
+    get:
+      operationId: listSolutionParameterGroups
+      tags:
+        - solution
+      summary: List all solution parameter groups
+      responses:
+        "200":
+          description: Parameter groups successfully listed
           content:
             application/json:
               schema:
                 type: array
                 items:
                   $ref: '#/components/schemas/RunTemplateParameterGroup'
         "400":
```

Comment:
The 400 response does not make sense on a list endpoint that has no request body